index.html

```diff
@@ -8,16 +8,48 @@
     <link rel="stylesheet" href="./style.css">
 </head>
   <body>
-    <h1>All Awesome books</h1>
-    <article id="book-list"> </article>   
-      <div class="bookInputs">
+    <header id="header">
+        <h1>Awesome Books</h1>
+        <ul>
+            <a href="#Books" id="books-link">My books</a>
+            <a href="#bookInputs" id="bookInputs-link">Add books</a>
+            <a href="#contact" id="contact-link">Contact</a>
+        </ul>
```

Comment:
**First of all great job acing it on your first attempt 👏**
- [ ] [OPTIONAL] Using `ul` without `li` disrupts the semantics and structure of the HTML document, and it may also cause accessibility and rendering issues. To maintain proper structure and accessibility, please make sure to always use `li` tags within `ul` or `ol`. ✔

- [ ] [OPTIONAL] As part of following the proper semantics It would also be better to wrap the `ul` inside a `nav` tag since the links inside are the pages navigation and It would provide better accessibility for devices like screen readers giving your website a better ranking in SEO overall.